utils/utils.go

```diff
@@ -83,3 +86,38 @@ func GetChildNumsFromHex(hexHash string) ([]uint32, error) {
 
 	return childNums, nil
 }
+
+// GenerateRandomSetOfKeys generates a random set of keys - xpriv, xpb and address
+func GenerateRandomSetOfKeys() (hdXpriv *bip32.ExtendedKey, hdXpub *bip32.ExtendedKey) {
```

Comment:
I would strongly think about two things:
1. maybe lets return the mnemonic, as this is commonly use in our reference backend (and other PoC(s) we created) 
2. maybe return struct with xPriv, xPub, Mnemonic, and error as a second returned value

Reply:
Done